### PR TITLE
Add friendly prompt for default value

### PR DESCRIPTION
**Pull request by larry801**
When I first used this wizard. The prompt for the default value confused me a lot. I wanted to delete the default value and enter my own. I added more text to the prompt to avoid confusion.
Please see issue #2 for detail.

### CODE REVIEW

File: NVDAAddonTemplate/__init__.py [hunk 1]
```diff
@@ -118,7 +118,8 @@ def getInput(prompt, default = ""):
 	answer = ""
 	while answer == "":
 		print(prompt)
-		answer = input(default)
+		default_prompt = "The default value is:(" + default + ")Please enter your choice"
```
Comment (derekriemer) [inline NVDAAddonTemplate/__init__.py]: Would something like this be a better choice?

Default: {default}
Please enter a value, or press enter to accept the default.

Also, please use a format string like this. it's easier if we decide to translate, and cleaner for changing later.

`		default_prompt = "The default value is: {default} Please enter your choice".format(default)`

**Review by derekriemer — CHANGES_REQUESTED** [review]
Thanks for the fix.  Please address this review comment and I'll make sure to submit this.

**Comment by larry801** [comment]
I have made requested changes and tested locally.

**Review by derekriemer — APPROVED** [review]

**Comment by derekriemer** [comment]
Thanks for doing this.

**Pull request merged**